nixos/modules/services/taler/libeufin.nix

```diff
@@ -0,0 +1,85 @@
+{
+  lib,
+  config,
+  pkgs,
+  ...
+}:
+let
+  this = config.services.taler.libeufin.bank;
+  talerEnabled = config.services.taler.enable;
+  dbName = "libeufin";
+  inherit (config.services.taler) configFile;
+in
+{
+  options.services.taler.libeufin.bank = {
+    enable = lib.mkEnableOption "GNU Taler libeufin bank";
+    package = lib.mkPackageOption pkgs "libeufin" { };
+    debug = lib.mkEnableOption "debug logging";
+  };
+
+  config = lib.mkIf (talerEnabled && this.enable) {
```

Comment:
```suggestion
  config = lib.mkIf (cfgt.enable && cfg.enable) {
```

Reply:
We're only using `this.enable` now ([here](https://github.com/NixOS/nixpkgs/pull/332699/files#diff-b0e1f52ea00abac3dcafdec2674afdee490e3e51c1084ee9b2aa5731b88b1690R27)).